src/desktop/assets/analytics.ts

```diff
@@ -19,16 +19,19 @@ mediator.on("all", (actionName: string, data?: object) =>
 if (sd.SHOW_ANALYTICS_CALLS) {
   // Log all pageviews
   window.analytics.on("page", function () {
+    // eslint-disable-next-line no-console
     console.info("ANALYTICS PAGEVIEW: ", arguments[2], arguments[3])
```

Comment:
I wonder if we could allow `console.info` and disallow `console.log`. Generally folks use `console.log` as a debugging tool. If we're doing something like `console.info` it seems a little more intentional. That'd have the added benefit of not having to do an eslint disable on these lines.